docs/examples/cloud_trace_exporter/README.rst

```diff
@@ -24,11 +24,19 @@ To use this exporter you first need to:
 
 .. code-block:: sh
 
+    cd opentelemetry-python/docs/examples/cloud_trace_exporter
     python basic_trace.py
 
 Checking Output
 --------------------------
 
 After running any of these examples, you can go to `Cloud Trace overview <https://console.cloud.google.com/traces/list>`_ to see the results.
 
-* `More information about exporters in general <https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html#configure-exporters-to-emit-spans-elsewhere>`_
\ No newline at end of file
+* `More information about exporters in general <https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html#configure-exporters-to-emit-spans-elsewhere>`_
```

Comment:
This link feels like it doesn't quite fit in the "Checking Output" section. Maybe it should go into a new "Further Reading" section?